.travis.yml

```diff
@@ -18,6 +18,7 @@ branches:
   only:
     - master
     - /support\/.*/
+    - feature/REPO-5118_Node_Download_Event
```

Comment:
Please remove all the temp Dev release stuff.

Reply:
Dev release stuff has been removed.